src/dreamer/trees/__init__.py

```
@@ -0,0 +1,64 @@
+    "LeafNode",
+    "InternalNode",
+    "RPTree",
+    "RPInternalNode",
+    "CoverTree",
+    "CoverNode",
+    "SklearnTreeWrapper",
+    "LSHSurprisal",
+    "GraphSurprisal",
+    "PrototypeSurprisal",
+    "create_tree",
+]
```

Ruff (0.14.10)

`__all__` is not sorted

Apply an isort-style sorting to `__all__`

(RUF022)